README.md

```diff
@@ -229,22 +239,25 @@ And you should see an output like this:
 
 
 ```
-Pose Score:  0.61885977
- nose                 x=210  y=152  score=1.0
- left eye             x=224  y=138  score=1.0
- right eye            x=199  y=136  score=1.0
- left ear             x=245  y=135  score=1.0
- right ear            x=183  y=129  score=0.8
- left shoulder        x=268  y=168  score=0.8
- right shoulder       x=161  y=172  score=1.0
- left elbow           x=282  y=255  score=0.6
- right elbow          x=154  y=254  score=0.9
- left wrist           x=236  y=333  score=0.7
- right wrist          x=163  y=301  score=0.6
- left hip             x=323  y=181  score=0.2
- right hip            x=191  y=251  score=0.0
- left knee            x=343  y=84   score=0.8
- right knee           x=162  y=295  score=0.0
- left ankle           x=318  y=174  score=0.1
- right ankle          x=167  y=309  score=0.0
+Inference time: 14 ms
+
+Pose Score:  0.60698134
+  NOSE                 x=211  y=152  score=1.0
```

Comment:
I'm confused about what made this omit the KeypointType part. I don't see an dictionary mapping keypoint items to their print.

Reply:
the enum class in python already have  a name member :)
https://docs.python.org/3/library/enum.html 

Calling that eliminates the need of mapping it to a name